packages/applications/ssr/src/components/pages/garanties-financières/actuelles/modifier/ModifierGarantiesFinancièresActuelles.form.tsx

```diff
@@ -46,7 +52,7 @@ export const ModifierGarantiesFinancièresActuellesForm: FC<
             }}
             iconId="fr-icon-arrow-left-line"
           >
-            Retour au détail des garanties financières
+            Retour aux détails des garanties financières
```

Comment:
le singulier est correct il me semble, et utilisé à pas mal d'endroit.

```suggestion
            Retour au détail des garanties financières
```